ppr-ui/src/composables/fees/enums/feeSummaryDefaults.ts

```diff
@@ -5,5 +5,7 @@ export enum FeeSummaryDefaults {
   DEFAULT_10 = 'feeDefault10',
   DEFAULT_500 = 'feeDefault500',
   SELECT_5 = 'feeSelect5',
-  SEARCH_5 = 'feeSearch5'
+  SEARCH_5 = 'feeSearch5',
+  SEARCH_8 = 'feeSearch8',
+  SEARCH_12 = 'feeSearch12'
```

Comment:
@doug-lovett Not entirely sure how/if these line up with the api and what is expected there.

Reply:
@cameron-freshworks No search fee info is exchanged between the UI and the API. The mhr api figures out the payment fling type codes and quantities from the token roles and the search selected matches.